src/freenet/node/NewPacketFormat.java

```diff
@@ -992,7 +992,7 @@ public void checkForLostPackets() {
 	 * queued at plus the lesser of half the RTT or 100ms if there are acks queued. 
 	 * Otherwise Long.MAX_VALUE to indicate that we need to get messages from the queue. */
 	@Override
-	public long timeNextUrgent(boolean canSend) {
+	public long timeNextUrgent(boolean canSend, long now) {
 		long ret = Long.MAX_VALUE;
```

Comment:
Do I understand it correctly that this changes the public API? Is it ensured that nothing uses this?